packages/app-project/package.json

```diff
@@ -59,6 +59,9 @@
     "swr": "~2.2.0",
     "validator": "~13.11.0"
   },
+  "peerDependencies": {
+    "@zooniverse/subject-viewers": "0.1.0"
+  },
```

Comment:
In FEM, [peerDependencies](https://nodejs.org/en/blog/npm/peer-dependencies#the-solution-peer-dependencies) is used when a library is imported by a parent app and that parent app already has a certain library in its package.json. For example, app-root is a Next.js app and therefore lib-user has a `peerDependency` of Next.js so it can use `next/link`.

`@zooniverse/subject-viewers` should be an actual dependency in app-project.

Reply:
A follow-up question to this -- where do you plan to use `@zooniverse/subject-viewers` in app-project? In the future maybe VolumetricViewer would be used for a subject's thumbnail or preview, but until then I don't see a need to install and import anything from `@zooniverse/subject-viewers` in app-project.

Reply:
This is a good catch - I had added this as part of getting the build to successfully build (I was seeing an error related to app-project) and didn't catch that it didn't help. This has been removed because the change to `ci-build.yml` is what fixed the build issue